python/pyngraph/ops/or.cpp

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+* Copyright 2017-2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "ngraph/op/or.hpp" // ngraph::op::Or
```

Comment:
Please fix `include` order.
System-headers before others. Add a blank line between system headers and other headers. 
```
#include <non-ngraph-file>

#include “ngraph” file
```